server/src/weaverbird/backends/pypika_translator/translators/base.py

```diff
@@ -551,11 +556,34 @@ def filter(
         columns: list[str],
         step: "FilterStep",
     ) -> StepContext:
+
+        # To handle relative date formats
+        if isinstance(step.condition, (ConditionComboAnd, ConditionComboOr)):
+            cond = step.condition
+
+            conditions = cond.and_ if isinstance(cond, ConditionComboAnd) else cond.or_
+
+            for sub_cond in conditions or []:
+                if isinstance(sub_cond, DateBoundCondition):
+                    if isinstance(sub_cond.value, RelativeDate):
+                        value_str_time = (
+                            evaluate_relative_date(sub_cond.value)
+                            .replace(tzinfo=timezone.utc)
+                            .strftime("%Y-%m-%d %H:%M:%S")
+                        )
+                        sub_cond.value = functions.Cast(value_str_time, 'TIMESTAMP')  # type: ignore
```

Comment:
out of curiosity, why's the `type: ignore` required here ?

Reply:
My linter was crying : "are you sure value is an attribute of sub_cond ?"

just checked and mypy don't care... so i removed it !